src/main/java/core/basesyntax/service/impl/FruitTransactionParserImpl.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitTransactionParser;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FruitTransactionParserImpl implements FruitTransactionParser {
+    private static final int OPERATION_INDEX = 0;
+    private static final int FRUIT_TYPE_INDEX = 1;
+    private static final int FRUIT_COUNT_INDEX = 2;
+    private static final String DATA_SPLIT_SYMBOL = ",";
+
+    @Override
+    public List<FruitTransaction> toTransaction(List<String> data) {
+        return data.stream()
+                .map(transaction -> transaction.toLowerCase().split(DATA_SPLIT_SYMBOL))

```

Comment:
```suggestion
                .map(transaction -> transaction.split(DATA_SPLIT_SYMBOL))
```
everything in input file already in lower case?